README.md

```diff
@@ -10,6 +10,11 @@ virtual machines are not desirable or possible (such as iOS, embedded targets),
 or where a developer is willing to produce a reasonably-sized self-contained program
 without the need to ship an additional execution runtime.
 
+Prerequesties:
+
+Linux:
```

Comment:
It's true for all platforms, not just Linux.

Reply:
Should I revise and submit (remove the Linux tag ) ?